README.md

```diff
@@ -60,3 +90,11 @@ Block blob "quickstart.txt" is deleted
 Container "demo" is deleted
 Done
 ```
+
+# Folder Introduction
+You will find the following folders: key-vault-node-quickstart-v3, which references the version 3.0 SDK and keyvault-node-quickstart-v4, which uses the 4.0 version of the SDK.
+* To use the latest Azure SDK version [azure-storage-js-v10-quickstart-v4] please add the following dependency:
+  * [@azure/storage-blob](https://www.npmjs.com/package/@azure/storage-blob)
+  * [@azure/abort-controller](https://www.npmjs.com/package/@azure/abort-controller)
+* For the previous stable Azure SDK version [azure-storage-js-v10-quickstart-v3] please add the following dependency:
```

Comment:
By using 'stable' here, you are insinuating that the latest one is not stable, but it was GA'd so it should be stable.

Remove 'stable'